package.json

```diff
@@ -21,8 +21,8 @@
     "e2e": "nx run-many --target=e2e --all",
     "affected:build": "nx affected:build",
     "affected:dev": "nx affected:dev",
-    "affected:lint": "nx affected:lint --base=develop",
-    "affected:test": "nx affected:test --base=develop",
+    "affected:lint": "nx affected:lint --base=develop --exclude=libs/dictionaries",
```

Comment:
Er det en spesiell grunn til å ikke linte libs/dictionaries?

Reply:
Ja, det har noe å gjøre med at JSON-filer har ikke gyldig JavaScript-syntaks i de fleste tilfeller. Tror denne type linting kan oppnås ved å ta i bruk tredjepartbiblioteker. Kan evt lage et issue på det

Reply:
Ah, ok. Er det mulig å legge til i en konfig-fil hvor vi kan ekskludere JSON-filer, eller å sette en konfig for eslint slik at den linter JSON-filer korrekt?

Reply:
Ja, det kan vi godt undersøke. Jeg lager et issue for det. HAr du mulighet til å skrive litt om under "Hvorfor vi trenger det" i issuet @terjesyl ?
https://github.com/Informasjonsforvaltning/fdk-frontend/issues/13